app/assets/javascripts/views/story_view.js

```diff
@@ -341,6 +342,42 @@ module.exports = FormView.extend({
     }
   },
 
+  cloneStory: function () {
+    this.cancelEdit();
+    var clonedStory = this.model.clone();
```

Comment:
const

Reply:
Opted to the use of the depreciated `var` instead of `let` or `const` to keep in line with the code style.

but yeah, going to swap for es6 convention.